docs/docs/nlu-training-data.mdx

```diff
@@ -3,92 +3,166 @@ id: nlu-training-data
 sidebar_label: NLU Training Data
 title: NLU Training Data
 description: Read more about how to format training data with Rasa NLU for open source natural language processing.
+abstract: NLU training data stores structured information about user messages.
 ---
 
-NLU training data is structured into different parts:
+The goal of NLU (Natural Language Understanding) is to extract structured information from user messages. This usually includes the user's [intent](glossary.mdx#intent) and any 
+[entities](glossary.mdx#entities) their message contains. You can
+add extra information such as **[regular expressions](#regular-expression-features)** and **[lookup tables](#lookup-tables)** to your
+training data to help the model identify intents and entities correctly. 
 
-* training examples
 
-* synonyms
+## Training Examples
 
-* regex features and
+NLU training data consists of example user utterances categorized by
+intent. 
+To make it easier to use your intents, give them names that relate to what the user wants to accomplish with that intent, keep them in lowercase, and avoid spaces and special characters. 
 
-* lookup tables
+:::note
+The `/` symbol is reserved as a delimiter to separate retrieval intents from response text identifiers. Make sure not
+to use it in the name of your intents.
 
-While training examples is the only part that is mandatory, including the others will help the NLU model
-learn the domain with fewer examples and also help it be more confident of its predictions.
+:::
 
-Synonyms will map extracted entities to the same name, for example mapping “my savings account” to simply “savings”.
-However, this only happens *after* the entities have been extracted, so you need to provide examples with the synonyms
-present so that Rasa can learn to pick them up.
 
-Lookup tables are lists of words used to generate
-case-insensitive regex patterns that are added to the regex features.
+## Entities
```

Comment:
Do we not have a quick section about intents?

Reply:
i'll resolve this for now as its the same structure as the training data format page